gui/WorldView.cxx

```diff
@@ -441,6 +473,126 @@ void WorldView::setBackgroundColor(QColor const& color)
 }
 
 //-----------------------------------------------------------------------------
+void WorldView::setActiveDepthMap(vtkMaptkCamera* camera, QString vtiPath)
+{
+  QTE_D();
+
+  if(d->UI.actionDepthmapDisplay->isChecked() && !d->depthmapAlreadLoaded)
+  {
+    std::string filename = vtiPath.toStdString();
+
+    vtkNew<vtkXMLImageDataReader> readerIm;
+
+    readerIm->SetFileName(filename.c_str());
+    readerIm->Update();
+
+    int width = readerIm->GetOutput()->GetDimensions()[0];
+    int height = readerIm->GetOutput()->GetDimensions()[1];
+
+    vtkNew<vtkPoints> points;
+
+    points->SetNumberOfPoints(width*height);
+
+    //Unprojecting the points from the vtifile
+
+    double dmToImageRatio = camera->GetImageDimensions()[0]/width;
```

Comment:
This looks suspicious; the operands are both `int`, so the result is an `int` promoted to a `double`. Is the ratio known to always be an integer?